InventoryManagementSystem/Product.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+namespace InventoryManagementSystem
+{
+	public class Product
```

Comment:
How the records of this entity are uniquely identified? by name? what if I have 2 different products items on reality but with the same name? should you pick another field to identify the records?